libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationViewListener.kt

```diff
@@ -129,4 +121,19 @@ abstract class NavigationViewListener {
      * Called when the info panel behavior updates to collapsed.
      */
     open fun onInfoPanelCollapsed() = Unit
+
+    /**
+     * Called when new instance of [MapView] is created.
+     *
+     * @param mapView the new [MapView] instance
+     */
+    open fun onMapViewAttached(mapView: MapView?) = Unit
+
+    /**
+     * Called when new instance of [MapView] is created. This callback can be used to remove
+     * listeners or subscriptions on the old [MapView] instance.
+     *
+     * @param mapView the old [MapView] instance
+     */
+    open fun onMapViewDetached(mapView: MapView?) = Unit
```

Comment:
```suggestion
    open fun onMapViewDetached(mapView: MapView) = Unit
```

Same here, only detach when there was a non-numm map attached